source/server/server.cc

```diff
@@ -468,6 +468,8 @@ RunHelper::RunHelper(Instance& instance, const Options& options, Event::Dispatch
       }) {
   // Setup signals.
   if (options.signalHandlingEnabled()) {
+// None of these signals exist on Windows
```

Comment:
Do you need a TODO here? How does Envoy know to shutdown on windows?

Reply:
There are multiple signaling mechanisms used in Windows including console signals, service control manager signals, custom events etc. Solving this is in our queue for the new development required, but out of scope for this particular catch-up PR.